bentoml/cli/yatai_service.py

```diff
@@ -68,11 +68,19 @@ def add_yatai_service_sub_command(cli):
         'compatible with Amazon S3, such as MinIO',
         envvar='BENTOML_S3_ENDPOINT_URL',
     )
+    @click.option(
+        '--base-url',
+        type=click.STRING,
+        default='.',
```

Comment:
Would this work with more complex path like `yatai/another_path`?

Lets add validation as well

Reply:
This option name might be confusing to users. Let's be more explict and name it something along the line of `--web-prefix-path`

Reply:
I tested with more complex paths, it does work